src/boot/measure.c

```diff
@@ -140,6 +142,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "version",     no_argument,       NULL, ARG_VERSION     },
                 { "linux",       required_argument, NULL, ARG_LINUX       },
                 { "osrel",       required_argument, NULL, ARG_OSREL       },
+                { "uname",       required_argument, NULL, ARG_UNAME       },
```

Comment:
so far we used the same order everywhere. in --help, in this table, in the enum.

Please don#t reorder this needlessly. i.e. find the best place for this concept an then put it in the same place in all lists that cover the same items.